lua/telescope/actions/init.lua

```diff
@@ -846,6 +846,12 @@ actions.remove_selected_picker = function(prompt_bufnr)
   end
 end
 
+--- Delete the word before the cursor.
+---@param prompt_bufnr number: The prompt bufnr
+actions.delete_word = function(prompt_bufnr)
+  vim.api.nvim_input "<c-s-w>"
```

Comment:
The only thing might be that we should make sure we're in insert mode here (otherwise I don't think it would do anything, right?)

Reply:
Can't we just add this mapping ?

```lua
        ["<c-w>"] = { "<c-s-w>", type = "command" },
```

The only question i have is what happens with 0.6.1

Reply:
Removed the action. You're right. It doesn't seem necessary.

As for 0.6.1, I'm not sure I have enough context to understand.

Reply:
I meant neovim release 0.6.1. dont worry i just tested it